scripts/release_scripts/update_changelog_and_credits.py

```diff
@@ -447,6 +453,67 @@ def create_branch(
             target_branch, current_release_version_number))
 
 
+def is_invalid_email_present(release_summary_lines):
+    """Returns new contributors and authors from the release
+    summary lines.
+
+    Args:
+        release_summary_lines: list(str). List of lines in
+            ../release_summary.md.
+
+    Returns:
+        list(str). A list of new contributors and authors.
+    """
+    authors_start_index = release_summary_lines.index(
+        release_constants.NEW_AUTHORS_HEADER) + 1
+    authors_end_index = release_summary_lines.index(
+        release_constants.EXISTING_AUTHORS_HEADER) - 1
+    new_authors = release_summary_lines[authors_start_index:authors_end_index]
+    contributors_start_index = release_summary_lines.index(
+        release_constants.NEW_CONTRIBUTORS_HEADER) + 1
+    contributors_end_index = release_summary_lines.index(
+        release_constants.EMAIL_HEADER) - 1
+    new_contributors = (
+        release_summary_lines[contributors_start_index:contributors_end_index])
+    new_email_ids = new_authors + new_contributors
+    return any(
+        release_constants.INVALID_EMAIL_SUFFIX in id
+        for id in new_email_ids)
+
+
+def get_release_summary_lines():
+    """Returns the lines from release summary file. It checks whether
+    incorrect email is present or ordering of sections is invalid.
+    In either case, the user will be asked to update the release
+    summary file and the lines will be re-read.
+    """
```

Comment:
Missing `Returns:` part of the docstring.

Reply:
Added.